static/app/views/ddm/focusArea.tsx

```diff
@@ -76,22 +76,27 @@ export function useFocusArea({
         brushType: 'rect',
       },
     });
-    isDrawingRef.current = true;
   }, [chartRef, hasFocusArea, isDisabled, onDraw]);
 
   useEffect(() => {
-    const handleMouseDown = event => {
-      const rect = chartRef.current?.ele.getBoundingClientRect();
+    const chartElement = chartRef.current?.ele;
+    const handleMouseDown = () => {
+      isDrawingRef.current = true;
+      startBrush();
+    };
 
-      if (isInRect(event.clientX, event.clientY, rect)) {
-        startBrush();
-      }
```

Comment:
@obostjancic it seems to work fine without this. Do you remember the reason behind this?

Reply:
I think there were cases around borders of the chart where clicking would activate the brush, but i could not reproduce it now.